File: Linguagem C/1011.c
#include <stdio.h>

int main()
{
    double R,VOLUME,pi;
    scanf("%lf",&R);
    pi=3.14159;
    VOLUME=(4.0/3)*pi*R*R*R;
    printf("VOLUME = %.3f\n",VOLUME);
    return 0;
}

File: Linguagem GO/1011.go
package main
 
import (
    "fmt"
)
 
func main() {
    
    var raio,volume float64
    fmt.Scanf("%f",&raio)
    
    volume = (4/3.0) * 3.14159 * (raio*raio*raio)
    
    fmt.Printf("VOLUME = %.3f\n",volume)
        
}
